spec/models/passenger_spec.rb

```diff
@@ -36,7 +36,18 @@
 
   describe 'temporary?' do
     it 'gets the not of permanent' do
-      expect(@passenger.temporary?).to eql true
+      expect(@passenger.temporary?).to be true
+    end
+  end
+
+  describe 'self.deactivate_expired_doc_note' do
+    it 'deactivates the expired passenger' do
+      create :doctors_note, passenger: @passenger,
+                            expiration_date: DoctorsNote.grace_period - 1.day
```

Comment:
Should use a mock here rather than actually calling a method this code depends on